libs/sdk-ui-filters/src/AttributeFilter/utils.ts

```diff
@@ -131,10 +131,15 @@ export function validateAttributeFilterProps(props: IAttributeFilterBaseProps) {
     if (selectionMode === "single" && filter) {
         invariant(
             isPositiveAttributeFilter(filter) && attributeElementsCount(filterAttributeElements(filter)) < 2,
-            "Provided 'filter' property is not compatible with given selectionMode. It needs to be positive filter with max one item selected in attribute elements",
+            "Provided 'filter' property is not compatible with given single selection mode. It needs to be positive filter with max one item selected in attribute elements",
         );
     }
 
+    invariant(
+        selectionMode !== "single" || (selectionMode === "single" && isEmpty(parentFilters)),
```

Comment:
This condition does not seems right.... 
`selectionMode !== "single"` is always false for single select, what is the use case?

Reply:
What about some variable aliases to make it more readable? Something like: `isSingleSelect && !hasParentFilters`